runtime/gc_modron_startup/mgcalloc.cpp

```diff
@@ -524,6 +524,29 @@ J9AllocateIndexableObject(J9VMThread *vmThread, J9Class *clazz, uint32_t numberO
 		/* The hook could release access and so the object address could change (the value is preserved).  Since this
 		 * means the hook could write back a different value to the variable, it must be a valid lvalue (ie: not cast).
 		 */
+
+#if defined(LINUX)
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+		GC_ArrayletObjectModel::ArrayLayout layout = extensions->indexableObjectModel.getArrayLayout((J9IndexableObject*)objectPtr);
+		if(extensions->indexableObjectModel.isDoubleMappingEnabled() && extensions->isVLHGC() && layout != GC_ArrayletObjectModel::InlineContiguous) {
```

Comment:
please add brackets around the `layout != GC_ArrayletObjectModel::InlineContiguous` clause